Calypso-Browser.package/ClyDataSourceItem.class/instance/includesActualObject..st

```diff
@@ -0,0 +1,4 @@
+testing
+includesActualObject: anObject
```

Comment:
Let's get with it and rename it later. Otherwise this PR will stay here for ever. Before there was an identity comparison. This improves what was done before by a lot, because this method can be overridden now in subclasses.